src/vm/builtins/builtin_runner/output.zig

```diff
@@ -24,14 +92,271 @@ pub const OutputBuiltinRunner = struct {
     /// # Returns
     ///
     /// A new `OutputBuiltinRunner` instance.
-    pub fn init(included: bool) Self {
+    pub fn from(included: bool, allocator: Allocator) Self {
```

Comment:
in turn to the `initDefault` suggestion, can this be just `init`?

Reply:
Agreed, done